docs/api/cloudrootfs.md

```diff
@@ -5,49 +5,53 @@ cloud rootfs will package all the dependencies refers to the kubernetes cluster
 ```shell script
 .
 ├── bin
-│   ├── conntrack
-│   ├── crictl
-│   ├── kubeadm
-│   ├── kubectl
-│   ├── kubelet
-│   └── seautil
+│   ├── conntrack
+│   ├── containerd-rootless-setuptool.sh
```

Comment:
why not put script in scripts dir?

Reply:
this is nerdctl script , nerdctl   download this scripts at same time